Source/FaceFX/Private/Audio/FaceFXAudioImplDefault.h

```diff
@@ -23,7 +23,7 @@ SOFTWARE.
 #include "FaceFXAudio.h"
 #include "Components/AudioComponent.h"
 
-class UActorComponent;
+#include "Components/AudioComponent.h"
```

Comment:
You are duping an include and removing a fwddec?

Reply:
didn't realize the include was added by facefx too. Thanks